source/configuration/read.txt

```diff
@@ -293,6 +293,55 @@ This partitioner is not compatible with hashed shard keys.
 
        **Default:** ``64``
 
+.. _spark-change-stream-conf:
+
+Change Streams
+--------------
+
+.. note::
+
+   If you use ``SparkConf`` to set the connector's change stream 
+   configurations, prefix ``spark.mongodb.change.stream.`` to each 
+   property.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 35 65
+
+   * - Property name
+     - Description
+
+   * - ``lookup.full.document``
+
+     - Determines what values your change stream returns on update 
+       operations.
+
+       The default setting returns the differences between the original 
+       document and the updated document.
+
+       The ``updateLookup`` setting returns the differences between the 
+       original document and updated document as well as a copy of the 
+       entire updated document.
+
+       .. tip::
+
+          For more information on how this change stream option works, 
+          see the MongoDB server manual guide 
+          :manual:`Lookup Full Document for Update Operation </changeStreams/#lookup-full-document-for-update-operations>`.
+
+       **Default:** "default"
+
+   * - ``publish.full.document.only``
+
+     - If ``true``, returns only the changed document instead of the 
+       full change stream document.
+
+       When set to ``true``, the connector automatically sets the 
```

Comment:
Issue: Let's keep the "true" descriptions parallel. So either stick with "if true" or "when set to true". Also, I think this part can be appended to the previous sentence instead of mentioning "if it is true" again.